remotebackup.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+#This script connects  to a remote server via ssh, creates a .tar.gz backup, and uses scp to secure copy it to the host computer. 
```

Comment:
I found the suggested format  in the comment, and  I'll make the necessary changes.  Thank you.